workflow/automation/lib/shared_template.py

```diff
@@ -133,6 +133,10 @@ def resolve_header(
         template_path = platform_config[const.PLATFORM_CONFIG.HEADER_FILE.name]
 
     j2_env = Environment(loader=FileSystemLoader(template_dir), trim_blocks=True)
+
+    from qcore import config
+    wallclock_limit = min(wallclock_limit, str(config.qconfig[config.ConfigKeys.MAX_JOB_WCT.name]))
```

Comment:
This shouldn't be needed as that should be covered by the confine_wct_node_parameters function